docs/extreme.md

```diff
@@ -67,3 +67,6 @@ buffering.
 In case a synchronous flush is needed, `dest.flushSync()` can be called.
```

Comment:
it seems to me that the `logger.flush` method is a footgun then, since `logger.flush` calls `flushSync` and not `flush` - with no warning about data loss in the docs.

1. Should we remove `logger.flush`
1. OR should we rename it to `logger.flushSync`
1. OR should we document the confusing nature of it

@mcollina @jsumners

Reply:
Can you check how much of this is done in #423? I’m definitely +1 to rename it to flushSync, and maybe also have a flush that’s async.